Flight/FlightManagement.cs

```diff
@@ -0,0 +1,275 @@
+﻿using AirportTicketBookingSystem.Enums;
+using AirportTicketBookingSystem.FileSystem;
+using AirportTicketBookingSystem.Exceptions;
+using System.Linq;
+
+namespace AirportTicketBookingSystem.Flights
+{
+    public class FlightManagement
+    {
+        public FlightManagement()
+        {
+            Flights = new List<Flight>();
+            PassengerFlights = new List<Flight>();
+            AddDefaultFlight();
+        }
+
+        public List<Flight> Flights { get; set; }
+        public List<Flight> PassengerFlights { get; set; }
+
+        public async Task BatchFlightUpload()
+        {
+            try
+            {
+                Console.WriteLine("Batch Flight Upload operation selected.");
+                Console.WriteLine("Enter CSV file Path");
+                string filePath = Console.ReadLine();
+                if (string.IsNullOrEmpty(filePath))
+                {
+                    throw new FlightManagementException("File path cannot be empty.");
+                }
+
+                var newFlights = await FileOperations.ReadFlightsFromCSVAsync(filePath);
+                Flights.AddRange(newFlights);
+
+                Console.WriteLine("Flights successfully uploaded.");
+            }
+            catch (FlightManagementException ex)
+            {
+                Console.WriteLine($"Flight management error: {ex.Message}");
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine($"An unexpected error occurred: {ex.Message}");
+            }
+        }
+
+        public void BookFlight()
```

Comment:
add the booked flight in a dynamic property for the object initiated is something not match the requirements in the Task, you should have a file for booked flights with flight number and passenger Id and other details for passenger booking